build.gradle

```diff
@@ -419,7 +444,8 @@ project('spring-integration-jpa') {
 	dependencies {
 		compile project(":spring-integration-core")
 		compile "org.springframework:spring-orm:$springVersion"
-		compile "org.eclipse.persistence:javax.persistence:$jpaApiVersion"
+//		compile "org.eclipse.persistence:javax.persistence:$jpaApiVersion"
+		compile ("org.hibernate.javax.persistence:hibernate-jpa-2.1-api:1.0.0.Final", optional)
```

Comment:
If we don't need `javax.persistence`, let's remove it altogether with its `jpaApiVersion` variable and change it to this `1.0.0.Final` for JPA-2.1!